test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -80,6 +86,7 @@ class SolidityExecutionFramework: public solidity::test::ExecutionFramework
 	bool m_compileViaYul = false;
 	bool m_compileToEwasm = false;
 	bool m_showMetadata = false;
+	bool m_metadataAppend = true;
```

Comment:
Keeping this as it is :)

Reply:
Why though? Wouldn't it make sense to change it to `m_appendCBORMetadata` here?

Reply:
Fixed it.